main.go

```diff
@@ -771,6 +771,7 @@ func checkNAT64Interface() error {
 	link, err := netlink.LinkByName(nat64If)
 	if err != nil {
```

Comment:
This is the change the issues was referring too, the interrumpted error is not necessary wrong, just indicates that there were more changes, but those changes may not be related to our link obtained here. This was the previous behavior of the library

```suggestion
	if err != nil && !errors.Is(err, netlink.ErrDumpInterrupted) {
```

Reply:
Good idea, I don't need to apply retry logic, I can just add this check to keep the behavior of the old version of netlink.